modules/ROOT/pages/requesting-access-to-a-portal.adoc

```diff
@@ -0,0 +1,19 @@
+= Requesting Access to a Portal for Salesforce Users 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Salesforce users can request portal access when they don't have Single Sign-On (SSO) configured. 
+
+Before you request access, confirm that you don't have a Single Sign-On (SSO) configuration. This can cause issues in the portal. For example, you might not see all APIs available to you if they are not available in the SSO configuration.
```

Comment:
I was a little confused by this paragraph. What causes issues? Requesting access if they don't already have SSO configured or confirming that they don't? 

I suggest changing "This can" to whichever scenario can cause issues to make it more clear.